bulk_grades/api.py

```diff
@@ -437,6 +437,38 @@ def get_rows_to_export(self):
                     row[f'grade-{block_id}'] = effective_grade
             yield row
 
+    def filtered_column_headers(self):
+        """
+        To trim down grade exports, only show subsections which were modified in a bulk update.
+        Returns: a filtered list of columns to export, preserving modified and non-subsection columns
+        """
+        unmodified_subsections = set()
+        for subsection in self._subsections.keys():
+            unmodified_subsections.add(subsection)
+
+        for row in self.result_data:
+            # Ignore rows which didn't introduce changes
+            if row['status'] == 'No Action':
```

Comment:
Nit: Is this string defined as a constant anywhere?

Reply:
No, it is set in situ in [super_csv/csv_processor.py::preprocess_file](https://github.com/edx/super-csv/blob/4eec510eb20dc07d90e8ee586cc2f86af45b602a/super_csv/csv_processor.py#L206)